tracer/src/Datadog.Trace/Logging/DirectSubmission/ImmutableDirectLogSubmissionSettings.cs

```diff
@@ -34,7 +33,7 @@ internal class ImmutableDirectLogSubmissionSettings
             string host,
             string source,
             IReadOnlyDictionary<string, string> globalTags,
-            Uri intakeUrl,
+            Uri? intakeUrl,
```

Comment:
Surely this shouldn't be null 🤔

Reply:
We take the value from `settings.DirectLogSubmissionUrl`, which is nullable